### DISCUSSION TITLE
레벨4 사용선 스쿼드 이슈 우선순위 논의

### DISCUSSION THREAD

**Discussion by gracefulBrown** [woowacourse/prolog#1003] [General]
https://github.com/orgs/woowacourse/projects/3/views/1

**Comment by verus-j** [comment]
#62 에디터 적용(이미지 삽입 기능)
#907 프롤로그 작성시 사진 URL 길이 수정
#932 프롤로그 글 작성이 되지 않는 버그를 확인한다.

**Comment by hanull** [comment]
이미지 저장, url 길이 
임시저장

**Comment by al-bur** [comment]
#765 임시저장
#794 검색기능 향상

**Comment by Yboyu0u** [comment]
#718. 임시저장 기능을 구현한다. 
#1001. 게시물, 댓글 작성 날짜를 N분전/N 시간전/N일전 형식으로 보여주기 
#882. 좋아요 누르기 쉽게 UI 변경 
#794. 검색을 할 경우 검색 키워드와 관련이 없는 내용도 검색되는 문제

**Reply by al-bur** [reply]
#1001 에 대한 시지프의 글

![image](https://user-images.githubusercontent.com/64825713/188383334-ba504c57-fc93-4cad-bf08-9baed04efbc7.png)

**Comment by JUDONGHYEOK** [comment]
1. 이미지 #62, 
2. 정렬기능 #807
